src/main/java/com/google/sps/servlets/Input.java

```diff
@@ -0,0 +1,69 @@
+package com.google.sps.servlets;
+
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Query;
+import com.google.cloud.datastore.FullEntity;
+import com.google.cloud.datastore.QueryResults;
+import com.google.cloud.datastore.StructuredQuery.OrderBy;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/input")
+public class Input extends HttpServlet {
```

Comment:
This is a servlet that provides a list of the donations in the datastore. `Input` isn't really a descriptive name for  the class. Can you think of something that better represents what the servlet does?

Reply:
Will do it. Thanks!